src/data/markdown/docs/30 xk6-browser/01 xk6-browser/02 BrowserContext.md

```diff
@@ -3,33 +3,24 @@ title: "BrowserContext"
-| <span title="Not implemented">❌</span> BrowserContext.route(url, handler[, options]) | Enables request interception for the provided URL to be processed by the provided handler function. |
-| [BrowserContext.setDefaultNavigationTimeout(timeout)](/javascript-api/xk6-browser/browsercontext/setdefaultnavigationtimeout) | Sets the default navigation timeout in milliseconds. |
-| [BrowserContext.setDefaultTimeout(timeout)](/javascript-api/xk6-browser/browsercontext/setdefaulttiontimeout) | Sets the default maximum timeout in milliseconds. |
-| <span title="Not implemented">❌</span> BrowserContext.setExtraHTTPHeaders(headers) | Sets extra headers that will be sent with every request initiated in the context. |
-| [BrowserContext.setGeolocation(geolocation)](/javascript-api/xk6-browser/browsercontext/setgeolocation) | Sets the context's geolocation. |
-| <span title="Deprecated">⚠</span> BrowserContext.setHTTPCredentials(httpCredentials) | Sets extra headers that will be sent with every request initiated in the context. **DEPRECATED**: create a new BrowserContext with httpCredentials instead. |
-| [BrowserContext.setOffline(offline)](/javascript-api/xk6-browser/browsercontext/setoffline) | Toggles the browser's connectivity on/off. |
-| <span title="Not implemented">❌</span> BrowserContext.storageState([options]) | Returns the storage state for this browser context, containing current cookies and local storage snapshot. |
-| <span title="Not implemented">❌</span> [BrowserContext.unroute(url[, handler])](/javascript-api/xk6-browser/browsercontext/unroute) | Removes a route created with [BrowserContext.route(url, handler[, options])](/javascript-api/xk6-browser/browsercontext/route). |
-| <span title="WIP: Requires async">🚧</span> [BrowserContext.waitForEvent(event[, optionsOrPredicate])](/javascript-api/xk6-browser/browsercontext/waitforevent) | Waits for the event to fire and passes its value into the predicate function. |
+
+| Method                                                                                                                                          | Description                                                                                                |
+|-------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| [BrowserContext.browser()](/javascript-api/xk6-browser/browsercontext/browser/)                                                                 | Returns the [Browser](/javascript-api/xk6-browser/browser) instance that this `BrowserContext` belongs to. |
+| <BWIPT id="442"/> [BrowserContext.clearCookies()](/javascript-api/xk6-browser/browsercontext/clearcookies/)                                     | Clears the `BrowserContext`'s cookies.                                                                     |
```

Comment:
We use the form "Clear" instead of the simple tense "Clears" in #731. Should we say "Clear the dadada" in this document as well?

Reply:
Resolved in: 4f66d1bb59ae9d11f3cdb9d500091a53473deec2